packages/jest-serializer-make-styles/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import { DEFINITION_LOOKUP_TABLE, RULE_CLASSNAME_INDEX } from '@fluentui/make-styles';
+
+export function print(val: string) {
+  const undesiredParts: string[] = [];
+  const lookupRegex = new RegExp(`${Object.keys(DEFINITION_LOOKUP_TABLE).join('|')}`, 'g');
+  let result: RegExpExecArray | null = null;
+  while ((result = lookupRegex.exec(val))) {
+    const [name] = result;
+    const [definitions] = DEFINITION_LOOKUP_TABLE[name];
+    const rules = Object.keys(definitions).map(key => `r?${definitions[key][RULE_CLASSNAME_INDEX]}`);
```

Comment:
I am not too familiar with makeStyles but are you trying to add the rtl prefix ? 

```suggestion
    const rules = Object.keys(definitions).map(key => `${RTL_PREFIX}?${definitions[key][RULE_CLASSNAME_INDEX]}`);
```